File: level1JS/calPrice.js
function solution(price, money, count) {
    console.log(price * count * (count + 1) / 2)
    const temp = price * count * (count + 1) / 2 - money;
    return temp > 0 ? temp : 0;
}

console.log(solution(3, 20, 4))

File: level1JAVA/src/CalPrice.java
class CalPrice {
    public long solution(int price, int money, int count) {
        return Math.max(((long) price * count * (count + 1)) / 2 - money, 0);
    }

    public static void main(String[] args) {
        CalPrice calPrice = new CalPrice();
        long result = calPrice.solution(3, 20, 4);
        System.out.println("result = " + result);
    }
}
